nilearn/plotting/img_plotting.py

```diff
@@ -1073,20 +1077,30 @@ def plot_glass_brain(stat_map_img,
     """
     if cmap is None:
         cmap = cm.cold_hot if black_bg else cm.cold_white_hot
+        # use only positive half of colormap if plotting absolute values
+        if plot_abs:
+            cmap = LinearSegmentedColormap.from_list(
+                'cmap_pos', cmap(np.linspace(0.5, 1, 256)),
+            )
 
     if stat_map_img:
         stat_map_img = _utils.check_niimg_3d(stat_map_img, dtype='auto')
         if plot_abs:
-            cbar_vmin, cbar_vmax, vmin, vmax = _get_colorbar_and_data_ranges(
-                _safe_get_data(stat_map_img, ensure_finite=True),
-                vmax=vmax,
-                symmetric_cbar=symmetric_cbar,
-                force_min_stat_map_value=0)
+            if vmin is not None and vmin < 0:
+                raise ValueError(
```

Comment:
Wondering if we should really raise en Error here ? A warning could be enough. Think of a loop creating many figures with `plot_abs` set to `True`, but a `vmin` calculated for each figure iteratively.

Reply:
Ok, I can force `vmin` to be `0` if it's negative and raise a warning instead of the error. Does that sound okay?

Reply:
Why force vmin to 0 ? The user may have his/her reasons to set it to a given value.

Reply:
Ok I see, that makes sense. Updated to allow arbitrary `vmin` value